socorro/processor/rules/breakpad.py

```diff
@@ -144,9 +144,10 @@ def __init__(
         command_line=(
             "timeout --signal KILL {kill_timeout} "
             "{command_path} "
-            "--raw-json={raw_crash_path} "
+            "--evil-json={raw_crash_path} "
             "--symbols-cache={symbol_cache_path} "
             "--symbols-tmp={symbol_tmp_path} "
+            "--no-color "
```

Comment:
It's irksome that we have these command line arguments in two places and kind of have to update both. Took me a while to figure out why the change I made didn't do anything.